modules/oe_content_entity/modules/oe_content_entity_contact/config/install/field.field.oe_contact.oe_general.oe_social_media.yml

```diff
@@ -13,7 +13,7 @@ bundle: oe_general
 label: 'Social media links'
 description: ''
 required: false
-translatable: false
+translatable: true
```

Comment:
Also, are the rest of the fields translatable in oe_content? It seems to me we only have to translate in EWCMS.